site-src/geps/gep-1426.md

```diff
@@ -146,6 +141,23 @@ This should be enforced by controller validation, which should set an `Accepted:
 
 The logic described here is a bit verbose, but could be validated with conformance tests, and is intended to allow an `HTTPRoute` configuring mesh traffic rules to additionally specify a `Gateway` `parentRef` (or `HTTPRoute` `parentRef` if [GEP-1058: Route Inclusion and Delegation](https://github.com/kubernetes-sigs/gateway-api/pull/1085) moves forward) to apply the same routing rules when exposing a mesh service outside the cluster.
 
+### Namespace boundaries
+
+In a mesh, routes can be configured by two personas:
+
+* Service producers, who want to modify behavior of *inbound* requests to their services
+* Service consumers, who want to modify behavior of *outbound* requests to other services.
+
+While these concepts are not directly exposed in the API, a route is implicitly fulfilling one of these rolls and behaves differently depending on the role.
+
+A route is a producer route when the `parentRef` refers to a service in the *same namespace*. This route SHOULD apply to all incoming requests to the service, including from clients in other namespaces.
+
+Note: Some implementations may only be able to apply routes on client-side proxies. As a result, these will likely only apply to requests from clients who are also in the mesh.
```

Comment:
How can we represent this when it comes to conformance? Do we need a new term for this?

Reply:
I would imagine conformance tests just are all about everything within the mesh. Otherwise, we have to define what is the behavior of out-of-mesh clients which is not something we should standardize IMO